chart/kubeapps/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: kubeapps
-version: 0.7.0
-appVersion: DEVEL
+version: 0.8.0
+appVersion: v1.0.0-beta.2
```

Comment:
revert this change, here it should be `DEVEL`

Reply:
👍